### PR TITLE
[Tizen Studio] Update Tizen-Core documentation

### PR DESCRIPTION
**Pull request by vivek-ellur**
Signed-off-by: Vivek Ellur/Things Platform /SRI-Bangalore/Staff Engineer/Samsung Electronics <vivek.ellur@samsung.com>

### Change Description ###

 - The documentation has been updated according to the latest Tizen-core commands.

### CODE REVIEW

**Review by omar-srbd — COMMENTED** [review]
Hello @vivek-ellur, apart from the correction/suggestions made on the file in this PR, there were some instances where I could not comment.
I am listing them below:
1.	Line 1 - “Tizen-Core command line interface commands”
Suggestion – “Tizen-Core Command Line Interface Commands”
Reason – Please use title case capitalization when writing first heading.
2.	Line 13 - “…Each workspace folder can contain main project along with the dependent projects. Workspace contain configuration yaml…”
Suggestion – “…Each workspace folder can contain the main project along with the dependent projects. Workspace contains configuration YAML”
Reason – An article “the” missing before the word “main”. The word “contain” should be plural here to agree with the subject. Since “YAML” is a specific language, I think it should be capitalized here. 
3.	Line 89 - "...excluded from build, format: pattern1,pattern2."
Suggestion - "...excluded from build, format: pattern, pattern2."
Reason - A space missing before the word "pattern2"
4.	Line 90 - "...included for test mode, format: pattern1,pattern2."
Suggestion - "included for test mode, format: pattern, pattern2."
Reason - A space missing before the word "pattern2"
5.	Line 92 - "...chrome simulator, format: arg1,arg2."
Suggestion -  "...chrome simulator, format: arg1, arg2."
Reason - A space missing before the word "arg2"
6.	Line 94 - “List of options for chrome inspector, format: arg1,arg2.”
Suggestion - “List of options for chrome inspector, format: arg1, arg2.”
Reason - A space missing before the word “arg2”.
7.	Line 97 -  “List of projects in the workspace and their dependencies, format: proj1= 
        [dep1:dep2:dep3],proj2.”
Suggestion - “List of projects in the workspace and their dependencies, format: proj1= 
        [dep1:dep2:dep3], proj2.”
Reason - A space missing before the word “proj2”.
8.	Line 101 – “To set llvm, arm and Release type for build, use following command:”
Suggestion – “To set llvm, arm and release type for build, use the following command.”
Reason – I am not sure why the word “release” has been capitalized here. An article “the” 
         missing before “following” and the line should end with a period.
9.	Line 116 – The alignment of this section should be indented to maintain consistency with the bullet point but it seems un-indented on the staging area
10.	Line 182 - “The command displays the list of project templates for all the versions of given workspace type.”
Suggestion – “The command displays the list of project templates for all the versions of the given workspace type.”
Reason – An article “the” missing before “given”.
11.	Line 198 - “List all the native application templates.”
Suggestion – “List of all the native application templates.”
Reason – Without the preposition “of”, the sentence sounds as a command, which doesn’t seem to be the purpose according to the context.
12.	Line 307 - “Option used to create samples (-s "Category_name,application_name").”
Suggestion – “Option used to create samples (-s "Category_name, application_name").”
Reason – A space missing before “application_name”.
13.	Line 350 - “This section describes how to list and create Native and Web Samples.”
Suggestion – “This section describes how to list and create native and web Samples.”
Reason – Is there any specific reason why the word “native” and “web” have been capitalized here?
14.	Line 438 - “Project dependencies, comma separated: -d proj1,proj2.”
Suggestion – “Project dependencies, comma separated: -d proj1, proj2.”
Reason – A space missing before the word “proj2”.
15.	Line 445 - “Following commands creates native BasicUI and ServiceApp in workspace and ServiceApp is added as dependent project of BasicUI.”
Suggestion – “Following command creates native BasicUI and ServiceApp in workspace and ServiceApp is added as dependent project of BasicUI.”
Reason – Missing articles in the line.
16.	Line 598 - “…Tizen Store or install the application to a Tizen device, you must generate a Tizen certificate.”
Suggestion – “…Tizen Store or install the application on a Tizen device, you must generate a Tizen certificate.”
Reason – The correct preposition should be "on" here.
17.	_Line 717, 723, 740, 746 - Code snippets should be preceded by a colon. So, please include a colon at the end of these lines._
18.	Line 870 - “...If there is a package file in the options(-b), the package is re-signed...The package is created for the project which is set as working folder as mentioned in the section [Set the working folder](#set-the-working-folder).”
Suggestion – “...If there is a package file in the option (-b), the package is re-signed...The package is created for the project which is set as a working folder as mentioned in the section [Set the working folder](#set-the-working-folder).”
Reason – The word “option” should be singular here.
19.	_Line 886, 887, 888 - Missing space after the word “[repack]” on these lines. So, please include a space after "[repack]"._
20.	Line 1051: Is there any reason why "Web Simulator" has been capitalized here?
21.	Line 1052: "| `-p`, `--project=STRING` | Specifies the Project to execute. |"
Suggestion: "| `-p`, `--project=STRING` | Specifies the project to execute. |"
_Is there any reason why the word "Project" has been capitalized here?_
22.	Line 1056 - “Run the application present in the specified workspace, on the emulator-26101.”
Suggestion – “Run the application present in the specified workspace on the emulator-26101.”
Reason – The comma is not necessary here.
23.	Line 1128 - "...After the workspace is initialized, use the following command to import the project:"
Suggestion: "...After the workspace is initialized, use the following command to import the project."
Reason: To maintain consistency throughout the content, the sentence should end with a period.
24.	_Line 91, 92, 93, 94, 95, 151, 152, 153, 154, 155 – In these lines “chrome” should be capitalized because chrome is a proper noun._

File: docs/application/tizen-studio/tizen-core/tizen-core-cli.md [hunk 1]
```diff
@@ -1166,12 +1113,12 @@ tz uninstall [options]
 
   Windows&reg;
```
[the remark this answers is not in the document]
Comment (vivek-ellur) [reply docs/application/tizen-studio/tizen-core/tizen-core-cli.md]: Updated as per review comments

File: docs/application/tizen-studio/tizen-core/tizen-core-cli.md [hunk 2]
```diff
@@ -34,47 +33,19 @@ tz init [options]
...
 
   Windows&reg;:
   ```
-  > mkdir C:\Users\native_workspace
-  > tz init -p mobile-6.5 -t native -w C:\Users\native_workspace
+  > mkdir C:\Users\workspace
+  > tz init -p mobile-6.5 -w C:\Users\workspace
   ```
 
   Ubuntu and Mac:
```
[the remark this answers is not in the document]
Comment (vivek-ellur) [reply docs/application/tizen-studio/tizen-core/tizen-core-cli.md]: Updated as per review comments

File: docs/application/tizen-studio/tizen-core/tizen-core-cli.md [hunk 3]
```diff
@@ -115,31 +86,31 @@ tz set [options]
...
   ```
-  $ tz set -A arm -c llvm -b release -W ~/native_workspace
+  $ tz set -A arm -c llvm -b release -w ~/workspace
   ```
 
-- Displays the list of all configurations for the given workspace. Individual options can be displayed by passing the required option flag.
+- Displays the list of all configurations for the given workspace. Individual options can be displayed by passing the required option flag:
 
 ```
 tz get [options]
```
Comment (omar-srbd) [inline docs/application/tizen-studio/tizen-core/tizen-core-cli.md:116]: @vivek-ellur,
Line 115: Please add two more spaces before the code tags (```). The correction suggestion on the line is not working properly. Hence, the comment. 
The change has been suggested to align the code with the bullet point. Please look into it. 
```suggestion
  tz get [options]

```

File: docs/application/tizen-studio/tizen-core/tizen-core-cli.md [hunk 4]
```diff
@@ -115,31 +86,31 @@ tz set [options]
...
-|  `-C`, `--chrome-path=STRING`                                      | Path of the chrome executable. |
-|  `-O`, `--chrome-simulator-options=CHROME-SIMULATOR-OPTIONS,...`   | List of options for chrome simulator, format: arg1,arg2. |
-|  `-M`, `--chrome-simulator-data-path=STRING`                       | Path of chrome simulator data. |
-|  `-L`, `--chrome-inspector-options=CHROME-INSPECTOR-OPTIONS,...`   | List of options for chrome inspector, format: arg1,arg2. |
-|  `-e`, `--chrome-inspector-data-path=STRING`                       | Path of chrome inspector data. |
+|  `-S`, `--src-file-patterns=SRC-FILE-PATTERNS,...`                 | Source file patterns excluded from build, format: pattern1, pattern2. |
+|  `-T`, `--test-file-patterns=TEST-FILE-PATTERNS,...`               | Source file patterns included for test mode, format: pattern1, pattern2. |
+|  `-C`, `--chrome-path=STRING`                                      | Path of the Chrome executable. |
+|  `-O`, `--chrome-simulator-options=CHROME-SIMULATOR-OPTIONS,...`   | List of options for Crome simulator, format: arg1, arg2. |
```
Comment (omar-srbd) [inline docs/application/tizen-studio/tizen-core/tizen-core-cli.md:92]: @vivek-ellur,
Please change "Crome" to "Chrome". Perhaps it was a typo.
```suggestion
|  `-O`, `--chrome-simulator-options=CHROME-SIMULATOR-OPTIONS,...`   | List of options for Chrome simulator, format: arg1, arg2. |

```

File: docs/application/tizen-studio/tizen-core/tizen-core-cli.md [hunk 5]
```diff
@@ -388,15 +360,16 @@ tz samples [options]
...
 | Option                 | Description                              |
 |------------------------|------------------------------------------|
-| `-W`, `--ws-dir`       | Specifies the workspace directory. |
+| `-w`, `--ws-dir`       | Specifies the workspace directory. |
+| `-t`, `--type=`       | Specifies the project type. |
 
 **Examples:**
 
-- List samples for mobile-6.5 native workspace.
+- List os samples for the mobile-6.5 workspace.
```
Comment (omar-srbd) [inline docs/application/tizen-studio/tizen-core/tizen-core-cli.md:368]: @vivek-ellur,
Please change "os" to "of". Perhaps it was a typo.
```suggestion
- List of samples for the mobile-6.5 workspace.

```

**Review by omar-srbd — COMMENTED** [review]
@vivek-ellur, Thanks for incorporating the changes. However, it seems like you have mistyped the corrections on a few occasions,
I have listed them below:

File: docs/application/tizen-studio/tizen-core/tizen-core-cli.md [hunk 6]
```diff
@@ -331,51 +303,51 @@ tz new [options]
...
   Ubuntu and macOS:
   ```
-  $ tz new -t TizenElmSharpApp -p elmsharp -w  ~/web_workspace
+  $ tz new -t TizenElmSharpApp -p elmsharp -T dotnet -w ~/workspace
   ```
 
 ## List and creation of Tizen samples
 
-This section describes how to list and create Native and Web Samples. Initialize the workspace as mentioned in the section [Initialize the Workspace](#initialize-the-workspace).
+This section describes how to list and create native and web Samples. Initialize the workspace as mentioned in the section [Initialize the Workspace](#initialize-the-workspace).
```
Comment (omar-srbd) [inline docs/application/tizen-studio/tizen-core/tizen-core-cli.md:350]: Change has been suggested to maintain consistency. Please look into it.
```suggestion
This section describes how to list and create native and web samples. Initialize the workspace as mentioned in the section [Initialize the Workspace](#initialize-the-workspace).

```

**Review by omar-srbd — COMMENTED** [review]
@vivek-ellur,
Thanks for incorporating the changes. However, there is one last issue that was missed out during the last check. I have put a comment on the issue.
Please look into it at your earliest convenience.

**Review by omar-srbd — APPROVED** [review]
@vivek-ellur, thank you for implementing the changes.

@mijin85cho @asm-foysal I am approving this PR for merging.

**Pull request merged**